hts.c

```diff
@@ -204,18 +205,19 @@ const char *hts_feature_string(void) {
 #endif
 
 #ifdef HAVE_LIBBZ2
-    "bzip2=yes ";
+    "bzip2=yes "
 #else
-    "bzip2=no ";
+    "bzip2=no "
 #endif
 
+    "htscodecs=%.40s";
+
+    snprintf(config, sizeof(config), fmt,
```

Comment:
Maybe stdlib's don't really define functions as macros any more so this doesn't really matter (and maybe there's already stuff like this in htslib, so this one doesn't matter either), but: `snprintf()` might be defined as a macro in which case you can't do `#ifdef` inside its argument list (C99 §6.10.3/11).

Yes, I really did actually get burnt by this once :smile:

Reply:
That's somewhat annoying....